### DISCUSSION TITLE
It's possible poll a relationManager Table after edit de ownerRecord?

### DISCUSSION THREAD

**Discussion by incloudsistemas** [filamentphp/filament#8510] [Help]
I have a ProductResource that I can add or change variant options to, and after that it updates another table of the variant's items. It works, but after updating, my relationshipmanager table does not live update, I need to refresh the page to see the update. Would there be a poll after the owner update solution?

**Comment by leandrocfe** [comment]
You can use livewire events to refresh the component.
V2 or V3?

**Reply by incloudsistemas** [reply]
Humm... Could you show me some doc or example of how to do it?

**Reply by leandrocfe** [reply]
YourRelationManager.php
```php
#[On('list-updated')]
public function updateList()
{
}
```
```php
$this->dispatch('list-updated'); // or $livewire->dispatch('list-updated')
```

**Reply by uwascan** [reply]
@leandrocfe Your answer works really well for v2. In V3, I ran into a situation where I was getting random errors because Relation Managers are Lazy loaded. They only load when you scroll them into view so when that event is fired and the target relation managers has not been loaded, you will end up with an error that basically says you cannot access the $table variable before initialization during rendering. The  backing $table variable in the Relation Manger has no default value.

**Reply by clinvest** [reply]
Hi did u find a solution for this? I also need to refresh a relation manager list from an action.

**Reply by leandrocfe** [reply]
Maybe using `$table->deferLoading(false)`

**Reply by emcarson** [reply]
Hi @uwascan, sorry to bother, but did you find a way to fix it? I have the exact same problem, I think it started when I upgraded to v3.
Thanks!
